google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingHandler.java

```diff
@@ -296,6 +300,16 @@ Formatter getFormatterProperty(String name, Formatter defaultValue) {
       }
       return Collections.emptyList();
     }
+
+    Synchronicity getSynchronicityProperty(String name, Synchronicity defaultValue) {
```

Comment:
Oh actually I didn't notice that this was in the context of a private class. You can just make this method (`getSynchronicityProperty`) package-private to be consistent with the rest.